ecosystem_cicd_tools/resources/plugins.json

```diff
@@ -3,17 +3,17 @@
         "description": "A Cloudify Plugin that provisions resources in AWS",
         "releases": "https://github.com/cloudify-cosmo/cloudify-aws-plugin/releases",
         "title": "AWS",
-        "version": "2.12.10",
-        "link": "http://repository.cloudifysource.org/cloudify/wagons/cloudify-aws-plugin/2.12.10/plugin.yaml",
+        "version": "2.12.11",
+        "link": "http://repository.cloudifysource.org/cloudify/wagons/cloudify-aws-plugin/2.12.11/plugin.yaml",
         "wagons": [
```

Comment:
Altarch wagons? same for the others

Reply:
This file I just keep everytime i make an update it is not actually used anywhere.